api/tools/index.ts

```
@@ -0,0 +1,27 @@
+import { BaasClient } from "@meeting-baas/sdk";
+import { McpServer } from "@modelcontextprotocol/sdk/server/mcp.js";
+import { z } from "zod";
+import { registerBotTools } from "./bots";
+
+export function registerTools(server: McpServer, apiKey: string): McpServer {
+  const baasClient = new BaasClient({
+    apiKey: apiKey,
+  });
+
+  // Register bot tools
+  server = registerBotTools(server, baasClient);
```

Biome (1.9.4)

[error] Reassigning a function parameter is confusing.

The parameter is declared here:

Use a local variable instead.

(lint/style/noParameterAssign)